view/scripts/view.js

```diff
@@ -95,11 +96,11 @@ export default class View {
     }
 
     renderTitle(title) {
-        return `<th>${title}</th>`;
+        return `<th>${he.encode(title)}</th>`;
     }
 
     renderEntry(entry) {
-        return `<td>${entry}</td>`;
+        return `<td>${he.encode(entry)}</td>`;
     }
 
     renderDeleteButton(id){
```

Comment:
should here the id not also be encoded with he?

Reply:
Unfortunately this did not work. But the id is generated automaticaly from the database and is not a user input, so I dont think that this will be an issue